### DISCUSSION TITLE
Wakeup display from save mode

### DISCUSSION THREAD

**Discussion by OK1FWG** [lora-aprs/LoRa_APRS_iGate#110] [Q&A] [answered]
Hi,
I am looking for a method how to wakeup display while in save mode (activated from the config). The save mode is working properly, but how to put it in active mode for a while?
The only reference to the dedicated override pin is found here in LoRa_APRS_iGate.cpp (L133-136):
```
  if (userConfig.display.overwritePin != 0) {
    pinMode(userConfig.display.overwritePin, INPUT);
    pinMode(userConfig.display.overwritePin, INPUT_PULLUP);
  }
```
But I didn't find any other reference which will handle with this pin. If the 'overwritePin' != 0, it just set the GPIO. What's next?
So, my question is, how was this meant? It seem to me it is not implemented yet?
I know, that the display is just informative and is woken up when sending beacon or receiving some data, but it can be good to check the display ie. for IP address, which is currently not displayed - btw, in my observation in the TaskManager.cpp in the function void StatusFrame::drawStatusPage(Bitmap &bitmap), the information from WiFiTask is not being read or this task from the list is ommited. That's why there is the IP address not displayed. I may be wrong, of course.
73, Willi

**Comment by peterus** [comment, marked as answer]
Hi Willi!
You are right! With the changes in the architecture this feature got lost and is currently not implemented. I created this bug ticket: #111 
For the IP issue: #103 
I will take care about this two tickets in in the next days!
73, Peter
